### DISCUSSION TITLE
Bug? "Are you sure you want to close this popup without saving?" when pressing ESC after opening a ticket card

### DISCUSSION THREAD

**Discussion by j-at-se** [cerb/cerb-release#81] [Q&A]
Steps to reproduce:
1. Go to a ticket worklist
2. Open the card for any ticket
3. Press ESC on your keyboard

Expected result:
1. The card is closed

Actual result:
1. The card is not closed
2. A confirmation pop up is displayed despite no changes being made

Screenshot:
![image](https://github.com/user-attachments/assets/c3d2923b-54d6-4f8b-b56c-2dbeb81f693f)

**Comment by jstanden** [comment]
Hi @j-at-se! Do you happen to have any hidden widgets or interaction toolbars on ticket cards? I'm able to reproduce it in some environments, but not on a clean 11.0.1 checkout.

I'll track it down. :)

**Reply by j-at-se** [reply]
Hi! My apologies for the long silence. No, not explicitly. Have you been able to determine the cause? There haven't been any further reports on our end so maybe it was a temporary, version-specific issue that got fixed on a newer update?
